VoxelSniperCore/src/main/java/com/github/kevindagame/brush/SplineBrush.java

```diff
@@ -202,7 +186,8 @@ public final void parseParameters(final String triggerHandle, final String[] par
                 return;
             }
             if (this.spline(this.endPts.get(0), this.endPts.get(1), this.ctrlPts.get(0), this.ctrlPts.get(1), v)) {
-                this.render(v);
+                this.initP(v);
+                this.actPerform(v);
```

Comment:
Are we calling the event here?

Reply:
https://github.com/KevinDaGame/VoxelSniper-Reimagined/commit/8ac84f931d408ce7fb4b957fa2cbe67c50fd0f15